src/components/discussion-card.jsx

```diff
@@ -26,12 +26,20 @@ export function DiscussionCard({ discussion }) {
 
   return (
     // Use API reponse to insert data
-    <div className="flex flex-col">
-      <p className="text-dark text-lg font-arialBold opacity-60">
+    <div className="flex flex-col" data-test-id="discussion-card-div">
```

Comment:
```suggestion
    <div className="flex flex-col" data-test-id={"discussion-card-"+discussion.id}>
```
if the top level div is unique than the component can be reused and always have a unique test id hierarchy.